test/models/item_test.rb

```diff
@@ -1,7 +0,0 @@
-require 'test_helper'
```

Comment:
To stop Rails from generating files in the `test/` directory, change the `require` line near the top of `config/application.rb` from:
```ruby
require 'rails/all'
```
to:
```ruby
# Pick the frameworks you want:
require "active_model/railtie"
require "active_job/railtie"
require "active_record/railtie"
require "action_controller/railtie"
require "action_mailer/railtie"
require "action_view/railtie"
require "sprockets/railtie"
# require "rails/test_unit/railtie"
```